seqr/views/utils/terra_api_utils.py

```diff
@@ -0,0 +1,186 @@
+# This module provides python bindings for the AnVIL Terra API.
+
+import json
+import logging
+
+from urllib.parse import urljoin
+
+from google.auth.transport.requests import AuthorizedSession
+from google.oauth2 import service_account
+
+from settings import SEQR_VERSION, TERRA_API_ROOT_URL, GOOGLE_SERVICE_ACCOUNT_INFO
+
+SEQR_USER_AGENT = "seqr/" + SEQR_VERSION
```

Comment:
I don't think our access header should be related to the seqr build version?

Reply:
I did it like this just because SCP did it like this.